Monika After Story/game/script-topics.rpy

```diff
@@ -6731,6 +6731,50 @@ label monika_song_dream:
     m 1ekbfa "My dream finally came true."
     return
 
+init 5 python:
+    addEvent(Event(persistent.event_database,eventlabel="monika_song_crime_and_punishment",category=['songs'],prompt="Crime and Punishment",pool=True, random=True))
```

Comment:
Like #3052, needs a TYPE_SHORT variant for random, needs to move to the new script file, also needs to use aff_range.